src/test/regress/sql/gp_orphaned_files.sql

```diff
@@ -0,0 +1,388 @@
+-- start_ignore
+create extension if not exists gp_inject_fault;
+-- end_ignore
+
+-- start_matchsubs
+-- m/ERROR:  Error on receive from seg\d+ slice\d+ \d+.\d+.\d+.\d+:\d+ pid=\d+: server closed the connection unexpectedly/
+-- s/ERROR:  Error on receive from seg\d+ slice\d+ \d+.\d+.\d+.\d+:\d+ pid=\d+: server closed the connection unexpectedly/ERROR:  Error on receive from segX sliceX X.X.X.X:X pid=X: server closed the connection unexpectedly/
+-- end_matchsubs
+
+select current_setting('data_directory') || '/' as dd
+\gset
+
+-- Check that orphaned files are not left on the coordinator when the files are
```

Comment:
As the feature is complex, we need to test it thoroughly and track very accurately that both requirements and code are covered by adequate test case. For the sake of traceability, please add some unique test case identifiers before each test case, at least some number, for ex.:

```suggestion
-- Test case 1.
-- Check that orphaned files are not left on the coordinator when the files are
```

Reply:
Done